src/App.js

```diff
@@ -1,25 +1,31 @@
-import logo from './logo.svg';
+import { useState } from 'react'
+import destinations from './destinations.json';
 import './App.css';
+import DestinationBox from './components/DestinationBox';
+
+
+export default function App() {
+
+  const [destinationData, setDestinationData] = useState(destinations)
+
+  function toggleVisited(name) {
+    setDestinationData(destinationData.map(destination => 
+      destination.name === name 
+      ? {...destination, visited: !destination.visited } 
+      : destination 
+    ))
+  }
+
+  function deleteDestination(name) {
+    setDestinationData(destinationData.filter(destination => destination.name !== name))
```

Comment:
try to improve indentation to make it more clear to read it

```
setDestinationData(
    destinationData.filter(
        destination => destination.name !== name
    )
)
```
or

```
const filteredDestinations = destinationData.filter(destination => destination.name !== name);
setDestinationData(filteredDestinations);
```

Reply:
1. Not sure what was going on with my default code formatter, but I fixed the configuration and now the indentation is correct when I use the keyboard shortcut again. 

2. Updated all of the code to reflect the ES6+ syntax for the functions like you suggested. Rana uses the long form in our live coding sessions in the mornings and tells us that we should practice writing everything the long way, so I've just been doing it because of that. 

3. For the DestinationBox, I think I must have misunderstood the instructions. They specify to render a list of DestinationBox components, so I  thought the syntax was correct the way that I wrote it? Or was it supposed to be the same <ul>{destinations.map....<li ...><DestinationBox props></li>} </ul> but in App.js? Bit confused on that one.